Client/Frontend/Settings/LoginDetailViewController.swift

```diff
@@ -380,7 +380,33 @@ extension LoginDetailViewController {
 
 // MARK: - Cell Delegate
 extension LoginDetailViewController: LoginDetailTableViewCellDelegate {
+    func textFieldDidEndEditing(_ cell: LoginDetailTableViewCell) { }
+    func textFieldDidChange(_ cell: LoginDetailTableViewCell) { }
+    
+    func canPeform(action: Selector, for cell: LoginDetailTableViewCell) -> Bool {
+        guard let item = infoItemForCell(cell) else {
+            return false
+        }
+
+        // Menu actions for password
+        if item == .passwordItem {
```

Comment:
oops, lost my comment! 

We could replace this with a `switch`.

Reply:
Done